server/index.js

```diff
@@ -34,6 +35,12 @@ function main() {
     this.set('Timing-Allow-Origin', '*');
     yield next;
   });
+
+
+  app.use(cors({
+    origin: true // do something more specific here
```

Comment:
`origin: true` allows all origins. My comment is a reminder to do something less permissive, eg whitelist training.blogs.ft.com and blogs.ft.com - wasn't immediately obvious from [koa-cors](https://github.com/evert0n/koa-cors) docs how to do this and I didn't want to waste time on it since you'd probably have an opinion on how to best handle this